src/Components/Server/src/ComponentHub.cs

```diff
@@ -209,6 +209,17 @@ public async ValueTask EndInvokeJSFromDotNet(long asyncHandle, bool succeeded, s
             _ = circuitHost.EndInvokeJSFromDotNet(asyncHandle, succeeded, arguments);
         }
 
+        public async ValueTask SupplyByteArray(long id, byte[] data)
+        {
+            var circuitHost = await GetActiveCircuitAsync();
+            if (circuitHost == null)
+            {
+                return;
+            }
+
+            await circuitHost.SupplyByteArray(id, data);
```

Comment:
@pranavkm it's fine if we don't await things here. The methods in `CircuitHost` should not throw exceptions and messages will be "serialized" as they enter the sync context. That said, I don't think awaiting here has any drawback other than slightly reduced performance.